src/domain/registration/form/partial/ChildFormField.tsx

```diff
@@ -57,54 +68,79 @@ const ChildFormField: React.FunctionComponent<ChildFormFieldProps> = ({
         </div>
 
         <div className={styles.childName}>
-          <EnhancedInputField
+          <Field
+            as={TextInput}
+            className={styles.formField}
             id={`children[${childIndex}].firstName`}
             name={`children[${childIndex}].firstName`}
             label={t('registration.form.child.firstName.input.label')}
             autoComplete="new-password"
-            component={InputField}
             placeholder={t(
               'registration.form.child.firstName.input.placeholder'
             )}
           />
-          <EnhancedInputField
+          <Field
+            as={TextInput}
+            className={styles.formField}
             id={`children[${childIndex}].lastName`}
             name={`children[${childIndex}].lastName`}
             autoComplete="new-password"
             label={t('registration.form.child.lastName.input.label')}
-            component={InputField}
             placeholder={t(
               'registration.form.child.lastName.input.placeholder'
             )}
           />
         </div>
 
-        <EnhancedInputField
+        <Field
+          as={TextInput}
+          className={styles.formField}
           id={`children[${childIndex}].postalCode`}
           name={`children[${childIndex}].postalCode`}
           label={t('registration.form.child.postalCode.input.label')}
           required={true}
-          component={InputField}
-          validate={validatePostalCode}
           placeholder={t(
             'registration.form.child.postalCode.input.placeholder'
           )}
+          invalid={
+            getIn(touched, `children[${childIndex}].postalCode`) &&
+            `children[${childIndex}].postalCode` !== undefined &&
+            validatePostalCode(`children[${childIndex}].postalCode`) !==
+              undefined &&
+            getIn(errors, `children[${childIndex}].postalCode`)
+          }
+          helperText={
+            getIn(touched, `children[${childIndex}].postalCode`) &&
+            t(getIn(errors, `children[${childIndex}].postalCode`))
+          }
         />
 
-        <EnhancedInputField
+        <Field
+          as={FormikDropdown}
+          className={styles.formField}
           id={`children[${childIndex}].relationship.type`}
           name={`children[${childIndex}].relationship.type`}
           label={t('registration.form.child.relationship.input.label')}
           required={true}
-          component={SelectField}
           options={getTranslatedRelationshipOptions(t)}
-          placeholder={t(
-            'registration.form.child.relationship.input.placeholder'
-          )}
+          onBlur={() =>
```

Comment:
The `<Field />` component you use inside `FormikDropdown` has about the same implementation as you have here. I know you have included it because `onBlur` didn't work, but don't forget to clean these out.